packages/cursorless-vscode-e2e/suite/recorded.test.ts

```diff
@@ -74,7 +73,8 @@ async function runTest(file: string, spyIde: SpyIDE) {
   const usePrePhraseSnapshot = false;
 
   const cursorlessApi = await getCursorlessApi();
-  const { graph, plainObjectToTarget } = cursorlessApi.testHelpers!;
+  const { graph, plainObjectToTarget, takeSnapshot } =
```

Comment:
note that `takeSnapshot` comes from extension return object now so that we don't need to put it in common, as it refers to `Target` and other big stuff from engine